marklogic/assets/dashboards/overview.json

```diff
@@ -0,0 +1,781 @@
+                "type": "query_value"
+            },
+            "id": 7180959663064248,
+            "layout": {
+                "height": 8,
+                "width": 15,
+                "x": 131,
+                "y": 22
+            }
+        },
+        {
+            "definition": {
+                "legend_size": "0",
+                "requests": [
+                    {
+                        "display_type": "line",
+                        "q": "sum:marklogic.requests.total_requests{$cluster}",
+                        "style": {
+                            "line_type": "solid",
+                            "line_width": "normal",
+                            "palette": "dog_classic"
+                        }
+                    }
+                ],
+                "show_legend": false,
+                "time": {},
+                "title": "Total number of active requests over $cluster",
+                "title_align": "left",
+                "title_size": "16",
+                "type": "timeseries",
+                "yaxis": {
+                    "include_zero": true,
+                    "label": "",
+                    "max": "auto",
+                    "min": "auto",
+                    "scale": "linear"
+                }
+            },
+            "id": 6497256550880508,
+            "layout": {
+                "height": 15,
+                "width": 38,
+                "x": 128,
+                "y": 53
+            }
+        },
+        {
+            "definition": {
+                "legend_size": "0",
+                "requests": [
+                    {
+                        "display_type": "line",
+                        "q": "sum:marklogic.forests.total_rate{$forest_name} by {forest_name}",
+                        "style": {
+                            "line_type": "solid",
+                            "line_width": "normal",
+                            "palette": "dog_classic"
+                        }
+                    }
+                ],
+                "show_legend": false,
+                "time": {},
+                "title": "Sum of the processing rate factors by $forest_name",
+                "title_align": "left",
+                "title_size": "16",
+                "type": "timeseries",
+                "yaxis": {
+                    "include_zero": true,
+                    "label": "",
+                    "max": "auto",
+                    "min": "auto",
+                    "scale": "linear"
+                }
+            },
+            "id": 1251925672919538,
+            "layout": {
+                "height": 15,
+                "width": 38,
+                "x": 73,
+                "y": 54
+            }
+        },
+        {
+            "definition": {
+                "background_color": "gray",
+                "content": "These information are available for all the databases included in `resource_filters`, so the most important ones can be monitored here",
```

Comment:
To make this clearer, let's make this a more direct statement about how to get database information to appear in this widget, with a link to the docs. Maybe something like, `Configure the databases Datadog will track using [resource filters](https://github.com/DataDog/integrations-core/blob/03831e4fb3f5d4e95f9aabe8d5a5da10dbf9547e/marklogic/datadog_checks/marklogic/data/conf.yaml.example#L60-L75)`